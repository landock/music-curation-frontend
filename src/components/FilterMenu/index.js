import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';
class FilterMenu extends PureComponent {
  render() {
    return (
      <div>
        <ul>
          <li>
            <button>All</button>
          </li>
          <li>
            <button>Tracks</button>
          </li>
          <li>
            <button>Artists</button>
          </li>
          <li>
            <button>Albums</button>
          </li>
          <li>
            <button>Label</button>
          </li>
        </ul>
        <div>
          <label>Dates</label>
          <input type="range" />
          <label>Country</label>
          <select />
          <label>City</label>
          <select />
        </div>
        <ul>
          <li>Release Type</li>

          <li>
            <button />
          </li>
          <li>
            <button>Tracks</button>
          </li>
          <li>
            <button>Artists</button>
          </li>
          <li>
            <button>Albums</button>
          </li>
          <li>
            <button>Label</button>
          </li>
        </ul>
      </div>
    );
  }
}

FilterMenu.propTypes = {};

export default FilterMenu;
