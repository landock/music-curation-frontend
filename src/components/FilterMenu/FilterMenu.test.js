
import React from 'react'
import { shallow, mount, render } from 'enzyme';
import renderer from 'react-test-renderer'
import FilterMenu from '.'

it('FilterMenu: default', () => {
  const component = renderer.create(<FilterMenu />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})