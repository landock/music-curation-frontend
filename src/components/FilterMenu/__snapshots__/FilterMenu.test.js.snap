// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterMenu: default 1`] = `
<div>
  <ul>
    <li>
      <button>
        All
      </button>
    </li>
    <li>
      <button>
        Tracks
      </button>
    </li>
    <li>
      <button>
        Artists
      </button>
    </li>
    <li>
      <button>
        Albums
      </button>
    </li>
    <li>
      <button>
        Label
      </button>
    </li>
  </ul>
  <div>
    <label>
      Dates
    </label>
    <input
      type="range"
    />
    <label>
      Country
    </label>
    <select />
    <label>
      City
    </label>
    <select />
  </div>
  <ul>
    <li>
      Release Type
    </li>
    <li>
      <button />
    </li>
    <li>
      <button>
        Tracks
      </button>
    </li>
    <li>
      <button>
        Artists
      </button>
    </li>
    <li>
      <button>
        Albums
      </button>
    </li>
    <li>
      <button>
        Label
      </button>
    </li>
  </ul>
</div>
`;
